.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "common"]
 	path = common
-	url = git@github.com:rtfd/common.git
+	url = https://github.com/rtfd/common
```

Comment:
With a `git@` URL is not possible to install this extension for the repository on Read the Docs.